Lombiq.Tests.UI/Constants/DirectoryPaths.cs

```diff
@@ -10,8 +10,11 @@ public static class DirectoryPaths
     public const string Temp = nameof(Temp);
     public const string Screenshots = nameof(Screenshots);
 
+    public static string GetTempDirectoryPath(params string[] subDirectoryNames) =>
+        Path.Combine(new[] { Environment.CurrentDirectory, Temp }.Concat(subDirectoryNames).ToArray());
+
     public static string GetTempSubDirectoryPath(string contextId, params string[] subDirectoryNames) =>
-        Path.Combine(new[] { Environment.CurrentDirectory, Temp, contextId }.Concat(subDirectoryNames).ToArray());
+        GetTempDirectoryPath(new[] { contextId }.Concat(subDirectoryNames).ToArray());
```

Comment:
I would argue, that as long as you don't reuse the `GetTempDirectoryPath` method anywhere, it's overhead to create two new arrays just to concat file paths.

Instead, you can stack `Path.Combine` calls:

```suggestion
        Path.Combine(Path.Combine(Environment.CurrentDirectory, Temp, contextId), Path.Combine(subDirectoryNames));
```

Reply:
Maybe formatted like this:
```suggestion
        Path.Combine(
            Path.Combine(Environment.CurrentDirectory, Temp, contextId),
            Path.Combine(subDirectoryNames));
```

Reply:
`GetTempDirectoryPath` is in use here: https://github.com/Lombiq/UI-Testing-Toolbox/blob/c69a65bd0906fd4f89236eb1d964b84f060a6b3f/Lombiq.Tests.UI/Extensions/FakeBrowserVideoSourceExtensions.cs#L14

But you are right stacking `Path.Combine` looks better.

Reply:
@0liver Please, if you consider this suggestion you have made here resolved, make the thread resolved aswell.